### ISSUE TITLE
Chrome Browser - Google Meet 

### ISSUE THREAD

**Issue by gitmyway** [dognotdog/ptpwebcam#14]
I could get this working on Firefox Browser and Zoom but not Chrome. In Google Meet is doesn’t show up. 

Maybe I am missing something ? Any settings to tinker with ? Did the usual allow camera settings in chrome.

**Comment by dognotdog** [comment]
@gitmyway I just tried this with Chrome 74.0.3729.169 on macOS 10.14, and it worked, but 84.0.4147.89 did no longer work as it it is signed with more restrictive entitlements.

Removing Chrome's codesigning via executing the following in the command line makes it see PTP Webcam:

```
killall -SIGKILL Chrome
codesign --remove-signature /Applications/Google\ Chrome.app/
codesign --remove-signature /Applications/Google\ Chrome.app/Contents/Frameworks/Google\ Chrome\ Framework.framework/Helpers/Google\ Chrome\ Helper.app
codesign --remove-signature /Applications/Google\ Chrome.app/Contents/Frameworks/Google\ Chrome\ Framework.framework/Helpers/Google\ Chrome\ Helper\ \(GPU\).app/
codesign --remove-signature /Applications/Google\ Chrome.app/Contents/Frameworks/Google\ Chrome\ Framework.framework/Helpers/Google\ Chrome\ Helper\ \(Plugin\).app/
```

**Comment by dognotdog** [comment]
Curiously, this seems to work only spuriously up to 1.0.0-beta3. I have just gone back and tried with all the releases back to -alpha8 and only that worked once, but not at other times.

Could this be a timing issue? The plugin takes a few seconds to advertise the video stream while it queries the camera, once the DAL plugin is loaded.

**Comment by dognotdog** [comment]
[v1.1.0-rc1](https://github.com/dognotdog/ptpwebcam/releases/tag/v1.1.0-rc1) should improve behaviore in Chrome.

**Comment by dognotdog** [comment]
It seems like at least on macOS 10.14, Chrome will not detect the camera if it has been connected before starting Chrome, but starting Chrome first, then connecting the camera seems to work reliably.

**Comment by wbrickner** [comment]
I'd like to reenable chrome codesignature after uninstallation (didn't work out for me), is this possible or do I need to do a clean install of chrome?

**Comment by dognotdog** [comment]
Yep, you need to reinstall Chrome. Unfortunately it hasn't been working on the latest Chrome releases under Big Sur :(
